app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/vaultunlocked/VaultUnlockedNavigation.kt

```diff
@@ -98,6 +99,9 @@ fun NavGraphBuilder.vaultUnlockedGraph(
                     passwordHistoryMode = GeneratorPasswordHistoryMode.Default,
                 )
             },
+            onNavigateToSetupUnlockScreen = {
+                navController.navigateToSetupUnlockScreen()
```

Comment:
Do we need a different route and get this added to the vault unlocked graph?

I think right now it is still using the destination from the top level graph and that could cause some problems down the line.